_posts/2023-02-20-geoserver-2-19-7-released.md

```diff
@@ -16,7 +16,7 @@ GeoServer [2.19.7](/release/2.19.7/) release is now available with downloads ([b
 
 This series has previously reached end-of-life, with an extra maintenance release being issued to address an urgent security vulnerability. Please apply this upgrade as a mitigation measure only. Upgrade to 2.22.x series for community support.
 
-Thanks to Andrea Amie (GeoSolutions) for makig this update available on behalf of GeoSolutions customers.
+Thanks to Andrea Amime (GeoSolutions) for making this update available on behalf of GeoSolutions customers.
```

Comment:
Aime :)

Reply:
Looking like I cannot spell my own name (when in a rush, at least!). Fixing.